CLAUDE.md

```
@@ -0,0 +1,119 @@
+# 覆盖率报告
+pnpm coverage
+
+# 单个测试文件
+pnpm vitest -- src/services/asyncCache.service.test.ts
+```
+
+## Architecture
+
+### 请求流程
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)